site/docs/v1/tech/core-concepts/_fusionauth_roles.adoc

```diff
@@ -0,0 +1,59 @@
+| `consent_deleter` | `631ecd9d8d404c13827780cedb8236fc` | Can delete consents.
+| `consent_manager` | `631ecd9d8d404c13827780cedb8236fd` | Can addd and edit consents.
+| `email_template_manager` | `631ecd9d8d404c13827780cedb8236e7` | Can add and edit email templates.
+| `event_log_viewer` | `631ecd9d8d404c13827780cedb8236fa` | Can view the event log.
+| `form_deleter` | `631ecd9d8d404c13827780cedb823702` | Can delete forms and form fields.
+| `form_manager` | `631ecd9d8d404c13827780cedb823703` | Can add and edit forms and form fields.
+| `group_deleter` | `631ecd9d8d404c13827780cedb8236f6` | Can delete groups.
+| `group_manager` | `631ecd9d8d404c13827780cedb8236f5` | Can add and edit groups.
+| `key_manager` | `631ecd9d8d404c13827780cedb8236fb` | Can add and edit keys.
+| `lambda_manager` | `631ecd9d8d404c13827780cedb8236f9` | Can add and edit lambdas.
+| `reactor_manager` | `631ecd9d8d404c13827780cedb8236ff` | Can add and edit reactor settings.
+| `report_viewer` | `631ecd9d8d404c13827780cedb8236e8` | Can view reports.
+| `system_manager` | `631ecd9d8d404c13827780cedb8236e9` | Can add and edit system configuration.
+| `tenant_deleter` | `631ecd9d8d404c13827780cedb8236f8` | Can delete tenants.
+| `tenant_manager` | `631ecd9d8d404c13827780cedb8236f7` | Can add and edit tenants.
+| `theme_manager` | `631ecd9d8d404c13827780cedb8236fe` | Can add and edit themes.
+| `user_action_deleter` | `631ecd9d8d404c13827780cedb8236f0` | Can delete user actions.
+| `user_action_manager` | `631ecd9d8d404c13827780cedb8236f1` | Can add and edit user actions.
+| `user_deleter` | `631ecd9d8d404c13827780cedb8236f2` | Can delete users.
+| `user_manager` | `631ecd9d8d404c13827780cedb8236f3` | Can add and edit users. 
+| `user_support_manager` | `631ecd9d8d404c13827780cedb823704` | Allows for a limited scope of user management. See below.
+| `user_support_viewer` | `631ecd9d8d404c13827780cedb823705` | Can view user information.
+| `webhook_manager` | `631ecd9d8d404c13827780cedb8236f4` | Can add or edit webhooks.
+
+|=== 
+
+The `user_support_manager` role is a role tuned for tier 1 technical support personnel and has a mix of capabilities. A user with such a role can:
+
+* Add a user.
+* Edit a user, except for any identity information that could be used to authenticate. For example, the email and username cannot be modified.
+* Add a registration with no role management. If a new registration is created it would receive the default roles only.
+* Edit a registration with no role modification.
+* Delete a registration. 
+* View a registration.
+* Send a forgot password request.
+* Send a verify email request.
+* Require a password change at next login.
+* Lock a user account.
+* Unlock a user account.
+* Modify 2FA settings if available.
+* Action a user.
+* Add a comment to a user.
+* Manage group membership.
+* Manage family membership.
+* Manage consents.
+* Manage sessions (refresh tokens).
```

Comment:
We can sync up on this one once the features is out and everything is concrete. Maybe we do one more review after the feature is out?

Reply:
Okay. This was cut and pasted from the issue, but let's defer releasing this until 1.23.0 is released and these values are firmed up.

Reply:
Oh, and I'll ask you to do another review, no problem.